### PR TITLE
Add roles for witness

### PR DESCRIPTION
**Pull request by babstar93**

### CODE REVIEW

File: data/.DS_Store
Comment (manimaarans) [inline data/.DS_Store:1]: Please remove unneccesary files @babstar93

File: .DS_Store
Comment (manimaarans) [inline .DS_Store:1]: Please remove this file @babstar93

**Review by manimaarans — CHANGES_REQUESTED** [review]

File: data/pg/ACCESSCONTROL-ROLEACTIONS/roleactions.json [hunk 1]
```diff
@@ -649,6 +649,24 @@
       "actionid": 2534,
       "actioncode": "",
       "tenantId": "pg"
+    },
+    {
+      "rolecode": "CASE_CREATOR",
```
Comment (manimaarans) [inline data/pg/ACCESSCONTROL-ROLEACTIONS/roleactions.json:654]: case editor should have role to add. remove and update a witness @babstar93

**Review by manimaarans — CHANGES_REQUESTED** [review]

File: data/pg/ACCESSCONTROL-ROLEACTIONS/roleactions.json (hunk 1 above)
Comment (babstar93) [reply data/pg/ACCESSCONTROL-ROLEACTIONS/roleactions.json:654]: We don't have a remove API. What do you mean by remove ? @manimaarans

**Review by manimaarans — APPROVED** [review]

**Pull request merged**